apps/meteor/app/api/server/v1/rooms.ts

```diff
@@ -387,7 +387,7 @@ API.v1.addRoute(
 				await findAdminRooms({
 					uid: this.userId,
 					filter: filter || '',
-					types: types || [],
+					types: types && !Array.isArray(types) ? [types] : undefined,
```

Comment:
Do you know if there's any case where `Array.isArray(types)` is truthy? And if there is, shouldn't we support that?

Reply:
There is an error here that should be fixed in another PR: when you send only one type (like ['p']), it comes along as a string, not inside an array.